packages/react-ui-kit/src/Form/DropFileInput.test.tsx

```diff
@@ -0,0 +1,177 @@
+/*
+ * Wire
+ * Copyright (C) 2022 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import React from 'react';
+import {matchComponent} from '../test/testUtil';
+import {DropFileInput, DropFileInputProps} from './DropFileInput';
+import {fireEvent, render} from '@testing-library/react';
+import {StyledApp, THEME_ID} from '../Layout';
+
+//note: we don't have to test file type validation using native file upload,
+// it won't allow to pick different type thant expected
+
+const getDefaultProps = () => ({
+  onInvalidFilesDropError: jest.fn(),
+  onFilesUploaded: jest.fn(),
+  headingText: 'Drag & Drop an image \nor',
+  labelText: 'select one from your device',
+  accept: 'image/png, image/jpeg',
+  description: 'Image (JPG/PNG) size up to 1 MB, minimum 200 x 600 px',
+});
+
+const ThemedDropFileInput = (props: DropFileInputProps) => (
+  <StyledApp themeId={THEME_ID.LIGHT}>
+    <DropFileInput {...props} />
+  </StyledApp>
+);
+
+describe('"DropFileInput"', () => {
+  it('matches snapshot', () => matchComponent(<ThemedDropFileInput {...getDefaultProps()} />));
+
+  it('returns file on native file upload', () => {
+    const file = new File(['(⌐□_□)'], 'chucknorris.png', {type: 'image/png'});
+
+    const props = {
+      ...getDefaultProps(),
+      accept: undefined,
+    };
```

Comment:
i dont necessarily understand why you chose to use a function for the default props and why you don't just spread them out in each component and add the necessary accept value. this seems like a little extra than is necessary